test/e2e/cpi_vm_test.go

```diff
@@ -0,0 +1,177 @@
+package e2e
+
+import (
+	"errors"
+	"k8s.io/klog/v2"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/vmware/govmomi/object"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// VMFolder is the folder that contains all virtual machines for CPV e2e testing
+const VMFolder = "/SDDC-Datacenter/vm/Workloads/clusterapi/"
+
+// getWorkerNode retrieves the worker node object for the E2E testing using workload cluster's clientset
+func getWorkerNode() (*corev1.Node, error) {
+	nodes, err := workloadClientset.CoreV1().Nodes().List(ctx, metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+	return getWorkerNodeFromList(nodes)
+}
+
+// getWorkerNodeFromList searches the first worker node that forms the cluster.
+// We assume all the name of worker node contains substring `-md-`
+func getWorkerNodeFromList(nodes *corev1.NodeList) (*corev1.Node, error) {
```

Comment:
Can you change to `getFirstWorkNodeFromList`

Reply:
sure